crates/ouroboros/src/praos/nonce.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2024 PRAGMA
```

Comment:
Super minor nitpick but shouldn't this be 2025 ?

Reply:
Can be both; I technically moved and entirely rewrote the nonce module that was created and licensed starting from 2024. So the 2024 terms still apply ^^"